symforce/test_util/backend_coverage_expressions.py

```diff
@@ -23,23 +23,23 @@ def backend_test_function(x: sf.Scalar, y: sf.Scalar) -> T.Tuple[sf.Scalar, ...]
     just the printer itself.
     """
     constants = (
-        sf.S.Zero,
-        sf.S.One,
-        sf.S.Half,
+        0.0,
+        1.0,
+        0.5,
```

Comment:
I converted these to floats because sympys rust backend will unfortunately format that as an integer which the rust compiler does not appreciate i.e. `0` is assumed to be an integer.

Reply:
These I think might cause issues, if you do actually try to print one of these things - I think the fix is to implement printing for these types that prints them as the correct floating point type, I think you can probably do this by implementing `_print_Integer` like we do in `ComplexCppCodePrinter`

Reply:
There is a `_print_Zero` defined in `StrPrinter` that I'm guessing you could override to do what you want

Reply:
The current version works for integers that aren't zero, right?  And then I think if you implement `_print_Zero` it'll work for zero also?  I might be wrong though.

We're similarly careful in C++ about making sure all literals (whether they're symbolic integers, floating point types, etc) are cast to the right type in the generated code - in C++ we sort of have the opposite problem, where in a lot of cases if you get it wrong, the code will compile but insert potentially expensive float<->double conversions